seven-days-to-dieconfig.json

```diff
@@ -1150,5 +1152,148 @@
             "False": "false",
             "True": "true"
         }
+    },{
+        "DisplayName":"Install Undead Legacy Stable",
+        "Category":"7d2d - Undead Legacy Settings",
+        "Description":"Auto-installs Undead Legacy Stable when the server is updated. NOTE: Set config file under \"SteamCMD and Updates\" to \"Undead Legacy Stable\" AND \"Release Stream\" to \"alpha20.5\"",
+        "Keywords":"install,UL,Undead,Legacy",
+        "FieldName":"UndeadLegacyStable",
+        "InputType":"checkbox",
+        "IsFlagArgument":false,
+        "ParamFieldName":"UndeadLegacyStable",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"false",
+        "EnumValues":{
+            "False": "false",
+            "True": "true"
+        }
+    },{
+        "DisplayName":"Install Undead Legacy Experimental",
+        "Category":"7d2d - Undead Legacy Settings",
+        "Description":"Auto-installs Undead Legacy Experimental when the server is updated. NOTE: Set config file under \"SteamCMD and Updates\" to \"Undead Legacy Experimental\"",
+        "Keywords":"install,UL,Undead,Legacy",
+        "FieldName":"UndeadLegacyExperimental",
+        "InputType":"checkbox",
+        "IsFlagArgument":false,
+        "ParamFieldName":"UndeadLegacyExperimental",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"false",
+        "EnumValues":{
+            "False": "false",
+            "True": "true"
+        }
+    },{
```

Comment:
You should be able to remove these two settings. Instead have the update stages rely on the value of `ConfigFile`. So if the user chooses that config file then it automatically downloads the mod files to match. :)

Reply:
I like this idea its one less step for the user to get confused about. Done